Blog-Section

georgioupanayiotis opened the pull request:
Added blogs section
- add blog card and filtering functionality with animations
-  replace static logo with CPSquadLogo component -update CSS variables for dark mode and add utility classes
- correct package.json formatting and dependencies

georgioupanayiotis committed:
Added blogs section
- add blog card and filtering functionality with animations
-  replace static logo with CPSquadLogo component
-update CSS variables for dark mode and add utility classes
- correct package.json formatting and dependencies

georgioupanayiotis commented:
<img width="1840" height="1191" alt="Screenshot 2025-10-22 at 12 03 03" src="https://github.com/user-attachments/assets/7d7a83b0-9e8f-41f1-8925-a5deaa79f6c2" />

UtsavSavani08 reviewed (changes requested):
@georgioupanayiotis On the current Blog page, clicking on any blog card or the 'Read More' button should open that particular blog with its full content. You can use dummy data for the blog content.

georgioupanayiotis committed:
Add blog data and implement blog post detail view

- Created a new file `blogData.js` containing an array of blog posts with detailed content, authors, dates, read times, categories, and tags.
- Updated `page.js` to include a new `BlogPost` component for displaying individual blog posts with a back navigation option.
- Modified `BlogCard` to handle reading more functionality, allowing users to view full blog content.
- Implemented state management for showing and hiding the full blog post view.
- Enhanced styling for blog content and navigation elements.

georgioupanayiotis commented:
please add label **hacktoberfest-accepted** it seems that your 
it still shows my PR as not accepted 


<img width="1192" height="208" alt="Screenshot 2025-10-22 at 17 56 30" src="https://github.com/user-attachments/assets/3ac1898c-b2de-45b1-9e34-ff4e80698386" />

UtsavSavani08 reviewed (changes requested):
@georgioupanayiotis  The Title section and the overall structure of the blog page should be as given image below 
<img width="1294" height="860" alt="image" src="https://github.com/user-attachments/assets/9aa1b555-564b-4bdc-81f6-5349a061470e" />

and also need to add animation as given in the refrence [website's](https://delta.nitt.edu/blogs) blog page.

The title section and overall whole page should look almost similar to the given refrence website.

georgioupanayiotis committed:
Refactor BlogCard and FilterButton components; add FilterDropdown for improved filtering; update BlogsPage layout and styles; include new background images.

UtsavSavani08 reviewed (changes requested):
@georgioupanayiotis The overall Layout and structure is almost perfect, But the main requirement of the blog title's background should be an animation not an image, which is not being fulfilled on the last chages. for the refrence you can visit the https://delta.nitt.edu/blogs and image attached below.

<img width="1312" height="442" alt="image" src="https://github.com/user-attachments/assets/536b0ff8-0195-4943-99e3-207623698e44" />

georgioupanayiotis committed:
Enhance BlogCard animations and add Marquee component for dynamic text display in BlogsPage

georgioupanayiotis commented:
@UtsavSavani08 i have made the changes

https://github.com/user-attachments/assets/a75230f9-906b-4c42-90a2-51109b65ff3c

UtsavSavani08 reviewed (changes requested):
@georgioupanayiotis  As per the video you have shared, The animation part is compelete can you just add some more styles in other words, Have fade like and font color and etc.. overall it must compeletly look like above refrence i have given.

UtsavSavani08 commented:
@georgioupanayiotis New changes have been made to the Home page — please pull the latest updates. Feel free to contact if you have any confusion or queries.

georgioupanayiotis committed:
Merge branch 'main' into Blog-Section

georgioupanayiotis committed:
Enhance BlogPost and Marquee components with improved styling and animations; update BlogsPage layout for better responsiveness and visual appeal.

DevSsChar commented:
@georgioupanayiotis the animation looks good, however the rounded background looks some what off, i request to make it more better, else the animation is fine. Mkae the background and text visually appealing
